airbyte-container-orchestrator/src/main/java/io/airbyte/container_orchestrator/ContainerOrchestratorApp.java

```diff
@@ -39,56 +48,129 @@
  * future this will need to independently interact with cloud storage.
  */
 @Slf4j
-public class ContainerOrchestratorApp {
+public class ContainerOrchestratorApp implements Runnable {
```

Comment:
curious, why implement Runnable? Did this already fit well with the Runnable interface?